python/lsst/daf/butler/datastores/chainedDatastore.py

```diff
@@ -377,12 +395,13 @@ def isDatasetAcceptable(dataset, *, name, constraints):
         # NotImplementedError being raised.
         allFailuresAreNotImplementedError = True
         for datastore, constraints in zip(self.datastores, self.datastoreConstraints):
+            okForChild: List[FileDataset]
```

Comment:
Ah, neat - I didn't know this syntax was possible.  I'm sure I'll use it in the future.